### ISSUE TITLE
Why TimberBlock not being used?

### ISSUE THREAD

**Issue by 16lkj** [DiffPitcher/DiffPitcher#8]
I noticed modules.py include:

class TimbreBlock(BaseModule):
    def __init__(self, out_dim):
        super(TimbreBlock, self).__init__()
        base_dim = out_dim // 4

        self.block11 = torch.nn.Sequential(torch.nn.Conv2d(1, 2 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(2 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.block12 = torch.nn.Sequential(torch.nn.Conv2d(base_dim, 2 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(2 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.block21 = torch.nn.Sequential(torch.nn.Conv2d(base_dim, 4 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(4 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.block22 = torch.nn.Sequential(torch.nn.Conv2d(2 * base_dim, 4 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(4 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.block31 = torch.nn.Sequential(torch.nn.Conv2d(2 * base_dim, 8 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(8 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.block32 = torch.nn.Sequential(torch.nn.Conv2d(4 * base_dim, 8 * base_dim,
                                                           3, 1, 1),
                                           torch.nn.InstanceNorm2d(8 * base_dim, affine=True),
                                           torch.nn.GLU(dim=1))
        self.final_conv = torch.nn.Conv2d(4 * base_dim, out_dim, 1)

    def forward(self, x):
        y = self.block11(x)
        y = self.block12(y)
        y = self.block21(y)
        y = self.block22(y)
        y = self.block31(y)
        y = self.block32(y)
        y = self.final_conv(y)

        return y.sum((2, 3)) / (y.shape[2] * y.shape[3])

But it is not being used, use_ref_t and   use_embed are all set to False, and there is no prepare_embed.py.
